sdk/storage/azure-storage-blob/tests/recordings/test_container.test_create_container_with_public_access_container.yaml

```diff
@@ -3,21 +3,21 @@ interactions:
     body: null
```

Comment:
Any reason you re-recorded this test? Probably can revert this?

Reply:
This comment applies to a couple of other tests in here too. If you did not change the test or the code the test is testing, no need to check in a new recording.

Reply:
I likely ran other tests outside of the one's I introduced and accidently added new recordings, will revert!